app/models/instance.rb

```diff
@@ -10,10 +10,13 @@
 class Instance < ApplicationRecord
   self.primary_key = :domain
 
+  attr_accessor :failure_days
+
```

Comment:
Finding this attribute yet nothing referencing it in this model is surprising, although I understand it's set in the controller for performance purposes…